common/tools/dev-tool/package.json

```diff
@@ -59,7 +59,7 @@
     "@types/concurrently": "^6.3.0",
     "@types/chai": "^4.1.6",
     "@types/chai-as-promised": "^7.1.0",
-    "@types/fs-extra": "^8.0.0",
+    "@types/fs-extra": "^9.0.13",
```

Comment:
Are you sure that these types actually correspond to fs-extra 10.x?

Reply:
Looking at the [changelog](https://github.com/jprichardson/node-fs-extra/blob/master/CHANGELOG.md#1000--2021-05-03) for fs-extra, I'm fairly confident that the 10.x changes won't have changed the type surface from 9.x. I think it should be fine.